src/cargo/core/package.rs

```diff
@@ -24,25 +25,26 @@ pub struct Package {
 
 #[derive(RustcEncodable)]
 struct SerializedPackage<'a> {
```

Comment:
Two other fields which I think should be included in here are perhaps source_id and package_id. The package id is the (name, version, source_id) triple, but it'd be good to pre-split-out the source and then have the package_id on-hand to cross reference to the encoded version of `Resolve` (which I think we'll deal with in a later PR)